Filter/BooleanFilter.php

```diff
@@ -32,23 +32,35 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
                     continue;
                 }
 
-                $values[] = ($v == BooleanType::TYPE_YES) ? 1 : 0;
+                $values[] = ($v == BooleanType::TYPE_YES) ? true : false;
```

Comment:
`($v == BooleanType::TYPE_YES)` is a condition. It always returns a boolean (either `true` or `false`). This line is basically saying : If the condition is `true`, store `true`, if it is `false` store `false`. I'm saying we could replace that with just "store the result of the condition"